ibtk/include/ibtk/PoissonSolver.h

```diff
@@ -65,9 +65,9 @@ class PoissonSolver : public virtual GeneralSolver
     PoissonSolver() = default;
 
     /*!
-     * \brief Destructor.
+     * \brief Default destructor.
```

Comment:
```suggestion
     * \brief destructor.
```

Reply:
Why not "default"?

Reply:
Writing 'default destructor.' like this makes it sound like, to me, there might be more than one destructor (just like how we label the default constructor). It seems odd but its not wrong.

Reply:
But it is literally the default destructor!

Reply:
You're right. I think this is just a personal problem I have...

Reply:
[cppreference](https://en.cppreference.com/w/cpp/language/destructor) also calls it the defaulted destructor, which I think is perverse.

Reply:
Thats the term I prefer :smile_cat: